logos/benchmarks/benchmark_logos.py

```
@@ -3191,259 +3144,75 @@ def _stop_ollama_docker_via_ssh(
-            print(f"  [ollama] '{model}' already cached — skipping pull.")
+        if model in deployed:
+            print(f"  [sllm] '{model}' already deployed — skipping.")
             continue
-        print(f"  [ollama] Pulling '{model}' (this may take a while) ...")
+        print(f"  [sllm] Deploying '{model}' (must be pre-converted with sllm-store save) ...")
         try:
-            with httpx.stream(
-                "POST",
-                f"{url.rstrip('/')}/api/pull",
-                json={"name": model, "stream": True},
+            result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)